app/library/Auth/Auth.php

```diff
@@ -153,12 +163,10 @@ public function loginWithRememberMe()
 
         $user = Users::findFirstById($userId);
         if ($user) {
-
             $userAgent = $this->request->getUserAgent();
             $token = md5($user->email . $user->password . $userAgent);
```

Comment:
Can we upgrade md5 to the bcrypt from the security component?

Reply:
Will do it in the next iteration, during mass tests creation, to make sure it works.